liberty-managed/JakartaEE9_README.md

```diff
@@ -0,0 +1,113 @@
+# Arquillian Liberty Managed with Jakarta EE 9
+
+An Arquillian container adapter (`DeployableContainer` implementation) that can start and stop a local Liberty process and run tests on it over a remote protocol (effectively in a different JVM). 
+
+## Prerequisites
+
+**Prerequisite Version**
+
+This `DeployableContainer` has been tested with the latest beta release of Open Liberty. Requires Jakarta EE 9.
+
+For Java EE 8 projects and below, check out the documentation [here](README.md).
+
+**Prerequisite Configuration**
+
+The following features are required in the `server.xml` of the Liberty server.
+
+```
+<!-- Enable features -->
+<featureManager>
+    <feature>jsp-3.0</feature>
+    <feature>localConnector-1.0</feature>
+    <feature>usr:arquillian-support-1.0</feature> <!-- Optional, needed for reliable reporting of correct DeploymentExceptions -->
+</featureManager>
+```
+
+Read more about configuring the `arquillian-support-1.0` feature [here](../liberty-support-feature/README.md).
+
+You will also need to enable the `applicationMonitor` MBean support:
```

Comment:
Maybe mention that it should be configured in server.xml?
"You will also need to enable the `applicationMonitor` MBean support in your server.xml"